pages/common/tlmgr-search.md

```diff
@@ -0,0 +1,24 @@
+# tlmgr search
+
+> Search for TeX Live packages using (Perl) regular expressions.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- Search for the package name and descriptions of all locally installed packages from a regular expression:
+
+`tlmgr search {{regular_expression}}`
```

Comment:
```suggestion
`tlmgr search "{{regular_expression}}"`
```

Optional fix: double quotes can be used to prevent POSIX-like shell from misinterpreting regex as glob.